From 393b113d8ddbd9e1b4282d5c2dd42818e189b28a Mon Sep 17 00:00:00 2001
From: Roger Bongers <r.l.bongers@gmail.com>
Date: Sun, 9 Jan 2022 16:53:16 -0500
Subject: [PATCH] Remove extra bar padding

---
 dwm.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/dwm.c b/dwm.c
index 5e4d494..1c2044e 100644
--- a/dwm.c
+++ b/dwm.c
@@ -705,7 +705,7 @@ drawbar(Monitor *m)
 	/* draw status first so it can be overdrawn by tags later */
 	if (m == selmon) { /* status is only drawn on selected monitor */
 		drw_setscheme(drw, scheme[SchemeNorm]);
-		tw = TEXTW(stext) - lrpad + 2; /* 2px right padding */
+		tw = TEXTW(stext) - lrpad /* + 2 */; /* 2px right padding */
 		drw_text(drw, m->ww - tw, 0, tw, bh, 0, stext, 0);
 	}
 
@@ -725,7 +725,7 @@ drawbar(Monitor *m)
 				urg & 1 << i);
 		x += w;
 	}
-	w = blw = TEXTW(m->ltsymbol);
+	w = blw = TEXTW(m->ltsymbol) - lrpad / 2;
 	drw_setscheme(drw, scheme[SchemeNorm]);
 	x = drw_text(drw, x, 0, w, bh, lrpad / 2, m->ltsymbol, 0);
 
-- 
2.34.1

